x-pack/test/functional/apps/spaces/enter_space.ts

```diff
@@ -32,7 +33,7 @@ export default function enterSpaceFunctionalTests({
         { space: 'another-space' }
       );
       const config = await kibanaServer.savedObjects.get({
-        id: await kibanaServer.version.get(),
+        id: stripVersionQualifier(await kibanaServer.version.get()),
         type: 'config',
```

Comment:
The only FTR test that was doing direct `config` access by ID using the SO client. Had to adapt for the version change described in previous comment.